### PR TITLE
added steps for bamCompare and heatmap

### PR DESCRIPTION
**Pull request by vivekbhr**

### CODE REVIEW

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 1]
```diff
@@ -237,7 +237,21 @@ We will now evaluate the quality of the immuno-precipitation step in the ChIP-se
...
+>
+> 1. Search for **bamCompare** under galaxy toolshed.
+> 2. Select the treatment sample, for example "Megakaryocyte_Tal1_R2.bam", as first sample, and the input sample "Megakaryocyte_Input_R2.bam" as second file.
+> 3. Check that you are using log2 ratio to compare the file, and click "Execute".
+> 
+> ![](../images/bamcom.png)
+>
+
+
+# Step 6: Determining Tal1 binding sites
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Why do you have twice "Step 6"?

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 2]
```diff
@@ -237,7 +237,21 @@ We will now evaluate the quality of the immuno-precipitation step in the ChIP-se
...
+
+> ### :pencil2: Hands-on: Generate Input-normalized bigwigs
+>
+> 1. Search for **bamCompare** under galaxy toolshed.
+> 2. Select the treatment sample, for example "Megakaryocyte_Tal1_R2.bam", as first sample, and the input sample "Megakaryocyte_Input_R2.bam" as second file.
+> 3. Check that you are using log2 ratio to compare the file, and click "Execute".
+> 
+> ![](../images/bamcom.png)
+>
+
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: `{: .hands_on}` is missing there

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 3]
```diff
@@ -337,6 +351,56 @@ We have processed ChIP-seq data from two stages of hematopoiesis and have lists
...
+> 5. From the output options, select "reference point".
+> 6. For region upstream and downstream of region file : Enter 5000 
+>
+> ![compMatrix](../images/compM.png)
+>
+> 7. Under advanced options , select "Yes" for "Convert missing values to zero" and "Skip zeros".
+> 8. Select "Execute"
+> 
+> ![computeMatrix](../images/compM2.png)
+
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: `{: .hands_on}` is missing there

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 4]
```diff
@@ -337,6 +351,56 @@ We have processed ChIP-seq data from two stages of hematopoiesis and have lists
...
+> 2. Select the computeMatrix output as an input.
+> 3. Under advanced options, go to "Labels for the samples ", and enter sample labels (in the order you added them in compueMatrix ), seperated by space.
+> 4. Click "Execute"
+>
+> ![plotHM](../images/plothm.png)
+>
+> The output shall look like this : 
+>
+> ![hm](../images/hm.png)
+
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: `{: .hands_on}` is missing there

**Comment by vivekbhr** [comment]
is it ok now?

**Comment by bebatut** [comment]
Thanks!

**Pull request merged**